lib/assessment.js

```diff
@@ -8,15 +8,19 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue, index, array) {
+  var minValue;
+  if (currentValue < minValue){
+    minValue = currentValue;
+  }
+  return minValue;
```

Comment:
Yep! 

Here's a more terse way to write it, taking advantage of the ternary operator.
```javascript
var min = function(a,b) {
 return (a < b)? a : b;
}
```